src/metabase/driver/postgres.clj

```diff
@@ -770,3 +773,49 @@
   ;; This could be incorrect if Postgres has been compiled with a value for NAMEDATALEN other than the default (64), but
   ;; that seems unlikely and there's not an easy way to find out.
   63)
+
+(defn- format-copy
+  [_clause table]
+  [(str "COPY " (sql/format-entity table))])
+
+(sql/register-clause! ::copy format-copy :insert-into)
+
+(defn- format-from-stdin
+  [_clause delimiter]
+  [(str "FROM STDIN NULL " delimiter)])
+
+(sql/register-clause! ::from-stdin format-from-stdin :from)
```

Comment:
I wanted to use the existing `:from`, but `(sql/format {... :from "STDIN"} ...)` of course gives you `["FROM ?" "STDIN"]`, which is no good. I don't think there's a way to treat the arg as pure SQL